helpers/handlebars/helpers/index.js

```diff
@@ -127,6 +128,12 @@ module.exports = {
 		}
 		return options.fn(this);
 	},
+	global: (globalVariable, options) => {
+		if (global[globalVariable] !== undefined) {
+			return global[globalVariable];
+		}
+		return options.inverse(this);
```

Comment:
there is no inverse for this, but return a default maybe?
this returns only a value, take care it does always...
https://handlebarsjs.com/guide/expressions.html#helpers

Reply:
register globals in res.locals instead, app.js#129 ff

Reply:
this seems to be ok, but shouldn't be called if there is no else option
https://stackoverflow.com/questions/33008106/typeerror-inverse-is-not-a-function-in-handlebars-helper